pyop2/types/dat.py

```diff
@@ -841,6 +844,9 @@ def vec_context(self, access):
         if access is not Access.READ:
             self.halo_valid = False
 
+    def increment_dat_version(self):
+        VecAccessMixin.increment_dat_version(self)
```

Comment:
Is this strictly necessary? I would have thought inheritance would have taken care of this.

Reply:
Turns out yes. The MRO causes us to hit the abstract method first.